pkg/event/target/webhook.go

```diff
@@ -131,5 +215,15 @@ func NewWebhookTarget(id string, args WebhookArgs) *WebhookTarget {
 				ExpectContinueTimeout: 2 * time.Second,
 			},
 		},
+		store: store,
 	}
+
+	if target.store != nil {
```

Comment:
Is this check needed, as of now `NewQueueStore` is never going to return a nil. Even if it does, it should be checked before not here.

Reply:
`target.store` will be `nil` if the `queueDir` is not configured

Reply:
i need this check here to kickstart the event store operations